apps/src/lib/kits/maker/BoardController.js

```diff
@@ -134,10 +134,9 @@ export default class BoardController {
     if (!this.board_) {
       return;
     }
-    // TouchSensors aren't in board_.register because they are our own wrapper
-    // around TouchPad, but we still need to reset them.
-    const touchSensors = _.filter(this.prewiredComponents, c => c instanceof TouchSensor);
-    this.board_.register.concat(touchSensors).forEach(BoardController.resetComponent);
+    this.board_.io.reset();
+    this.board_ = null;
+    this.prewiredComponents = null;
```

Comment:
this looks a lot more like a `destroy` method now than a reset; where do we expect these values to be reinitialized?